hw03-generics/src/main/java/homework/CustomerReverseOrder.java

```diff
@@ -0,0 +1,18 @@
+package homework;
+
+import java.util.LinkedList;
+
+@SuppressWarnings("java:S1319")
+public class CustomerReverseOrder {
+
+    public final LinkedList<Customer> customers = new LinkedList<>();
```

Comment:
1) private
```suggestion
    private final LinkedList<Customer> customers = new LinkedList<>();
```

2) Слева тип лучше объявлять интерфейсом и наиболее узким. Какой здесь подойдет интерфейс? Остальной код менять не надо - там всё хорошо.